src/osip-simulation-controller/src/main/resources/Bundle.properties

```diff
@@ -14,3 +14,17 @@ opcua.pipe.flowRate=Flow rate
 opcua.motor=Motor
 opcua.motor.rpm=Rotations per minute
 opcua.pipe.in=Incoming pipe
+controller.helpdialog.title=Help
+monitoring.helpdialog.closeButton=Close
+controller.helpdialog.mainTab.header=General
+controller.helpdialog.mainTab.introduction=The OSIP production site simulation simulates a production site consisting of three upper tanks containing liquids which get filled through some pipes into a mix tank. It is possible to adjust a motor speed, temperatues and valves.
+controller.helpdialog.mainTab.layoutSection=Layout
+controller.helpdialog.mainTab.layoutText=The simulation has three tanks on top which get liquid from pipes above of them. Each tank has a valve above and below the tank to set the input and output flow rates. Additionally there is a temperature sensor attached to each tank. From each of those tanks a pipe is connected to the mix tank which mixes the liquids of the upper tanks. The mix tank is a regular tank without a valve above and with a mix motor in it.
```

Comment:
Additionally **[Komma]** there is a temperature sensor attached to each tank. From each of those tanks **[Komma]** a pipe is connected to the mix tank which mixes the liquids of the upper tanks. The mix tank is a regular tank without a valve above and with a mix motor in it.